modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase

```diff
@@ -138,38 +138,10 @@ definition {
 				zipCode = "12.345-67");
 		}
 
-		task ("Then The employee user is Redirection to page Organization") {
-			EVPNavigator.assertActiveBreadcrumb(activeItem = "Organization");
-		}
-	}
-
-	@description = "LPS-162894 - Verify that the user can create a new organization and a 'Thank you' message will be displayed"
-	@priority = 5
-	test CanseeSuccessMessageAfterCreateOrganization {
-		task ("Given that the Employee user go to organization page") {
-			EVPOrganization.goToCreateOrganization();
-		}
-
-		task ("When the Employee user create an organization") {
-			EVPOrganization.fillOrganizationForm(
-				bankingInfo = "Bank Test",
-				city = "City Test",
-				contactEmail = "test@liferay.com",
-				contactName = "Contact Test",
-				contactPhone = 12345,
-				country = "Country Test",
-				done = "true",
-				organizationName = "Organization Test 1",
-				organizationSite = "www.test.com.br",
-				smallDescription = "Description Test",
-				state = "State Test",
-				street = "Street Test",
-				taxId = 12345,
-				zipCode = "12.345-67");
-		}
-
-		task ("Then can see the message of successfully is displayed") {
+		task ("Then The Employee user view a 'Thank you' message and is Redirection to page Organization") {
```

Comment:
Just a typo here, could you change this task to: `"Then The Employee user view a 'Thank you' message and is Redirected to page Organization"`